.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "cpu/cc26xx/lib/cc26xxware"]
 	path = cpu/cc26xx/lib/cc26xxware
 	url = https://github.com/g-oikonomou/cc26xxware.git
+[submodule "dev/ca8210"]
+	path = dev/ca8210
+	url = git@github.com:abhijit-mahajani-imgtec/ca8210-contiki.git
```

Comment:
This will need to be changed once https://github.com/FlowM2M/ca8210-contiki/pull/1 is merged.